app/javascript/dashboard/components/widgets/WootWriter/FullEditor.vue

```diff
@@ -95,6 +97,24 @@ export default {
     openFileBrowser() {
       this.$refs.imageUploadInput.click();
     },
+    async handleImageUpload(url) {
+      try {
+        const fileUrl = await this.$store.dispatch(
+          'articles/uploadExternalImage',
+          {
+            portalSlug: this.$route.params.portalSlug,
+            url,
+          }
+        );
+
+        return fileUrl;
+      } catch (error) {
+        useAlert(
+          this.$t('HELP_CENTER.ARTICLE_EDITOR.IMAGE_UPLOAD.UN_AUTHORIZED_ERROR')
```

Comment:
This will always show Unauthorized error in all cases, should we show a generic error?

Reply:
I took the inspiration from google doc. Do you have any other suggestions?
![CleanShot 2024-09-10 at 11 25 10](https://github.com/user-attachments/assets/74f7b0cc-87d3-43a4-ad79-54f8768eb27c)